Moda.Web/src/moda.web.reactclient/src/app/components/common/timeline/moda-timeline.tsx

```diff
@@ -18,27 +18,28 @@ import {
   ModaDataGroup,
   ModaDataItem,
   ModaTimelineOptions,
-  RangeItemTemplateProps,
 } from '.'
 import { ModaEmpty } from '..'
 import { getLuminance } from '@/src/utils/color-helper'
+import { ItemTemplate } from '@/src/app/components/common/timeline/types'
+import { ThemeName } from '@/src/app/components/contexts/theme/types'
 
 const { Text } = Typography
 
-export type ModaTimelineProps<TI = any, TG = any> = {
-  data: ModaDataItem<TI>[]
-  groups?: ModaDataGroup<TG>[]
+export type ModaTimelineProps<T> = {
+  data: ModaDataItem<T>[]
+  groups?: ModaDataGroup<T>[]
```

Comment:
The 'T' on the item and group can be different.  For the roadmap timeline, they will likely be the same, but for the objectives and a few other future ones it's not hierarchical in the same way.  The parents (groups) are different types compared to the items.

Reply:
Added the typing for groups back in